lib/components/artist/articles/article.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View, Text } = React;
+
+import colors from '../../../../data/colors';
+import SerifText from '../../text/serif';
+import ImageView from '../../opaque_image_view';
+
+class Article extends React.Component {
+  render() {
+    const article = this.props.article
+    return (
+      <View style={{width: 300, marginBottom: 40}}>
```

Comment:
Btw, I believe the design calls for something like `width of artist view - 60pt`, so we need to remove these hardcoded values, but it might require some layout trickery, so maybe it’s best if you create a ticket for that and assign it to.